test/tests/fvbcs/fv_neumannbc/tests

```diff
@@ -0,0 +1,28 @@
+[Tests]
+  [fv_neumann]
+    design = 'FVNeumannBC.md'
+    issues = '#16477'
+    type = 'Exodiff'
+    input = 'fv_neumannbc.i'
+    exodiff = 'fv_neumannbc_out.e'
+    requirement = 'The system shall run a simple 1D diffusion problem with a Neumann BC value.'
+  []
+  [fvbcs_internal]
+    design = 'FVNeumannBC.md'
+    issues = '#16882'
+    type = RunException
+    input = 'fv_neumannbc.i'
+    cli_args = "FVBCs/inactive=''"
+    expect_err = 'A FVFluxBC is being triggered on an internal face with centroid'
+    requirement = 'The system shall error out if FVFluxBC, in this case a FVNeumannBC, are used internally.'
```

Comment:
We try to keep implementation details like object names out of requirements. So maybe `FVFluxBC` -> `finite volume flux boundary condition` and `FVNeumannBC` -> `Neumann boundary condition`.